Exploration Queries/IoT/Process_byIoTDevice.yaml

```diff
@@ -0,0 +1,24 @@
+﻿Id: 1f3ecde7-5c69-4d44-ac93-5feac6d1cd2f
+DisplayName: Most frequent command executions on the device
+Description: Most frequent command executions on the device
+InputEntityType: IoTDevice
+InputFields:
+  - DeviceId
+  - IoTHub
+OutputEntityTypes:
+  - Process
+QueryPeriodBefore: 30m
```

Comment:
Just verifying 30 minutes is the right amount of time for the amount of data available?

Reply:
I randomly chose a time frame, as I understood these fields are practically meaningless right now, and aren't used for anything (to my understanding, same thing has been made in other queries as well).

Reply:
Yeah, for exploration they for right now, but I wanted to confirm 30m is the right time frame you recommend if that functionality is used with exploration queries.